src/carousel/carousel.ts

```diff
@@ -53,6 +61,11 @@ export class NgbCarousel implements AfterContentChecked,
   @Input() interval: number = 5000;
   @Input() wrap: boolean = true;
 
+  @HostBinding('tabIndex')
```

Comment:
We don't need `@HostBinding` for constants.... we could achieve the same effect in `host: {tabIndex: 0}`. Having said this I'm not sure that `0` is the best value. Maybe we should `-1` instead? Or ask users to provide the value?

I must say that I don't have enough knowledge on tab indexes to answer this question myself...